week1/homework/src/server.js

```diff
@@ -10,11 +10,38 @@ function createServer(port) {
 
   const server = http.createServer((request, response) => {
     // TODO: Write your homework code here
+    switch (request.url) {
+      case '/state':
+        response.setHeader('content-Type', 'application/json');
+        response.write(JSON.stringify({ state }));
+        break;
+      case '/add':
+        response.setHeader('content-Type', 'application/json');
+        state++;
+        response.write(JSON.stringify({ state }));
+        break;
+      case '/subtract':
+        response.setHeader('content-Type', 'application/json');
+        state--;
+        response.write(JSON.stringify({ state }));
+        break;
+      case '/reset':
+        response.setHeader('content-Type', 'application/json');
+        state = 10;
+        response.write(JSON.stringify({ state }));
+        break;
+      default:
+        response.statusCode = 404;
+        let message = 'Not found';
```

Comment:
Better to use `const` instead of `let` for declaration if the variable is no re-assigned.

Reply:
thanks for the nice comment's .I'll work on it next times